build/embed.bzl

```diff
@@ -231,11 +248,19 @@ def _cc_embed(ctx):
 cc_embed = rule(
   implementation = _cc_embed,
   attrs = {
-       "data" : attr.label_list(
+       "data": attr.label_list(
            allow_files = True,
            mandatory = True,
            doc = "Targets to turn into strings and embed in the binary."
        ),
+       "namespace": attr.string(
+           default = "embedded",
```

Comment:
nit: I think users should always(?) specify a namespace. Is it possible to make the default more obvious, for example "unspecified_namespace"

Reply:
well, I think the common use for this rule is to embed static data, eg, a .txt file, a .gif, ...

I can mark this attribute as mandatory, and force the user of the rule to always specify a namespace, or assume that the common case is to embed some data - a handful of files - and it's ok to have a default namespace?

I'd leave as is unless you feel strongly.